src/redux/reducer.ts

```diff
@@ -827,22 +827,22 @@ export const reducer = createReducer(initialState, (builder) => {
         }
         if (action.newLines) {
             newMapEquipments.lines = action.newLines;
-            // @ts-expect-error TODO: set parameter(s) optional in diagram-viewer
+            // @ts-expect-error TODO: set parameter(s) optional in network-viewer
             newMapEquipments.completeLinesInfos();
```

Comment:
maybe you can pass an empty array ? 
```suggestion
            newMapEquipments.completeLinesInfos([]);
```